python/vtool/ramen/rigs_maya.py

```diff
@@ -286,21 +306,26 @@ def sub_color(self, color):
         self.rig.attr.set('sub_color', color)
 
         if in_maya:
-            pass
-            # for control in self._sub_controls:
-            #    control.color = color
+            for control in self._controls:
+                subs = attr.get_multi_message(control, 'sub')
+                for sub in subs:
+                    control_inst = Control(sub)
+                    control_inst.color = color
 
     @property
     def shape(self):
-        return self.rig.attr.get('shape')
+        return self.rig.attr.get('shape')[0]
```

Comment:
I'm planning to be able to pass a list of shape names.    Currently rig controls build over a sequence of joints.   I'm making most data iterable so that I can interpolate over the sequence  in the future.     I'll add some docstrings to explain it once I'm sure I want to keep it.